docs/odcs.rst

```diff
@@ -46,46 +50,49 @@ information for requesting a compose to be created by ODCS::
     compose:
       # Optional, default False. Inherit compose used for building parent image
       inherit: true
-      # Optional, default "tag".
-      type: tag
-      # Required for "tag" type. Must contain at least one item.
+      # Required for ODCS Koji "tag" type usage. Must contain at least one item.
       packages:
       - package-1-name # This is a package name, not an NVR.
       - package-2-name
-      # Optional. Default value is configured in OSBS environment
-      sigkeys:
-      - AB123456
+      # Optional. Default and possible values are configured in OSBS environment
+      # which will be a mapping of intent -> list of signing keys
+      #     release -> my-release-key
+      #     beta -> my-beta-key, my-release-key
+      #     unsigned -> <empty>
+      intent: release
 
 Alternatively, for modules support::
```

Comment:
Not having 'type', and inferring it from the presence of either 'packages' or 'modules', *seems* OK. As long as it can be described by a JSON schema, I'm fine with doing it that way, and I think it can ("oneOf").